UNRELEASED_CHANGELOG.md

```diff
@@ -1,17 +1,77 @@
 ## Common changes for all artifacts
+### 🐞 Fixed
+
+### ⬆️ Improved
+
+### ✅ Added
+
+### ⚠️ Warning
```

Comment:
What would we use "Warning" for? Is this for deprecations only? Could use a better name.

Reply:
Deprecations or maybe some internal behavior changed.
Do you suggest another name?

Reply:
Maybe we can steal "Changed" from the Swift changelog 😄